### ISSUE TITLE
Feature suggestions.

### ISSUE THREAD

**Issue by ThomasIO** [NamelessMC/Nameless#249]
Add bios, emojis, following more similar to twitter, friends more similar to facebook, and make an option for custom pages to link to another other website or another link within the webserver.

**Comment by jesseke55** [comment]
No just no

**Comment by samerton** [comment]
- Bios: Custom profile fields will be added in the future, allowing you to create things such as bios.
- Emojis: The current smileys in the post editor are most likely the closest to emojis Nameless will have.
- Following and Friends: Implementation of a following system has already started, however following someone actually doesn't do anything yet as it's still being worked on. If you want to switch from the Friends system (which is enabled by default) to the Followers system, you can check "Enable Followers" in the AdminCP -> Core tab.
- Custom pages links: This is also planned and will hopefully be added in the near future.

**Comment by ryancito02** [comment]
Support for bitcoin wallets in store?

**Comment by samerton** [comment]
@ryancito02 The store itself is handled by either Buycraft or Minecraft Market, as a result the ability to use Bitcoin would be down to them.

**Comment by PugaBear** [comment]
For custom link pages, you can create your own by following these steps:

1. Create a custom page via the admin panel. Name it what you want, and put some placeholder text in the page content. 

2. Go to the 'pages' directory, and create a new file named '_name-of-custom-page_.php'. Put this code into that file:
```
<?php
header("Location: http://website.link/here/");
exit;
?>
```

All done, enjoy. :)

**Comment by samerton** [comment]
Update:

Complete:
- Bios: custom profile fields implemented in v2
- Emojis: implemented in v2
- Custom page links: implemented in v1.0.11

Todo:
- Following/friends: a full follower implementation coming in v2

**Comment by Derkades** [comment]
https://github.com/NamelessMC/Nameless/issues/853
